docs/source/llms/gateway/deprecation.rst

```diff
@@ -1,6 +1,100 @@
 .. _gateway-deprecation:
 
+=============================
 MLflow AI Gateway Deprecation
 =============================
 
-TODO
+The MLflow AI Gateway is deprecated and has been replaced by the MLflow deployments API.
+This page describes how to migrate from the MLflow AI Gateway to the MLflow deployments API.
```

Comment:
@harupy Can we add a link to the MLflow Deployments page / the page where we're going to talk about MLflow Deployments + LLMs? cc @BenWilson2

Reply:
@harupy I think we should have a page that says "MLflow Deployments for LLMs" or something like that

Reply:
> I think we should have a page that says "MLflow Deployments for LLMs" or something like that

Sounds good. Let's add it first in a separate PR, then update this PR.

Reply:
Thanks!